ui/app/templates/components/transit-key-action/datakey.hbs

```diff
@@ -3,7 +3,7 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<form onsubmit={{action @doSubmit (hash param=@param context=@context nonce=@nonce bits=@bits)}}>
+<form onsubmit={{perform @doSubmit (hash param=@param context=@context nonce=@nonce bits=@bits)}} ...attributes>
```

Comment:
I tested this, and the problem is I want to use concurrency because it allows me to use waitFor and also lets me use `isRunning` which I use for a loading state. If don't use preform I hit ember issues trying to run a concurrency task. But maybe there is a trick I don't know about?

Thoughts?